Example/GroupsExample.php

```diff
@@ -41,6 +41,9 @@ public function groupsExample()
         $createGroupResponse = $group->create('Test');
         $newGroupKey = $createGroupResponse->getGroupKey();
         
+        // Obtain list of all groups
+        $currentGroups = $group->list();
```

Comment:
Example should be something like:

```
$listGroupsResponse = $group->list();
$groups = $listGroupsResponse->getGroups();
```

Also add typehint e.g.

`/** include correct var above too, see other examples */`